docs/source/tutorials-and-examples/tutorial.rst

```diff
@@ -78,13 +78,13 @@ First, train a linear regression model that takes two hyperparameters: ``alpha``
 
     .. code-block:: py
 
-        python examples/sklearn_elasticnet_wine/train.py
+        python sklearn_elasticnet_wine/train.py
```

Comment:
Can we insert a comment like this?

```suggestion
        # Make sure the current working directory is 'examples'
        python sklearn_elasticnet_wine/train.py
```

Reply:
@harupy : Makes sense. In fact that's what exactly happened to me 😂

Reply:
@harupy : Added. Thanks for the suggestions.

Reply:
Thanks for the quick fix :)